src/main/java/com/revature/validation/JsonValidation.java

```diff
@@ -258,5 +272,65 @@ public void validateQuestionPoolExists(Integer qpId) {
                     "Question with id " + qpId + " does not exist",null);
         }
     }
+    
+    public void validatePersonFields(Person person){
+    	
+    	List<Integer> roleIds = new ArrayList<>();
+		List<PersonRole> roleList = personRoleDao.findAll();
+		System.out.println(roleList);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)